routes/messages.routes.js

```diff
@@ -16,7 +16,26 @@ router.post("/", (req, res) => {
     MessageAppService
         .saveMessage(destination, body)
         .then(response => res.json(response.data))
-        .catch(err => res.json(err))
+        .catch(err => {
+            
+            if(!err.config.data.includes("destination") && !err.config.data.includes("body") ){
```

Comment:
Why do you check the data after you send ?
Why do you use err.config.data?
Is this some axios magic can you explain?